zerver/views/webhooks/trello/templates.py

```diff
@@ -0,0 +1,2 @@
+TRELLO_SUBJECT_TEMPLATE = '{board_name}.'
```

Comment:
I think this should be `u'{board_name}'`. Or you can use `from __future__ import unicode_literals` at the top of this file to make all string literals in this file unicode by default.